executor/bind.go

```diff
@@ -83,6 +88,17 @@ func (e *SQLBindExec) dropSQLBind() error {
 	return err
 }
 
+func (e *SQLBindExec) dropSQLBindByDigest() error {
+	if !e.isGlobal {
```

Comment:
For safety, `e.sqlDigest` cannot be empty. Please returning an error here if `e.sqlDigest==""`.